geomstats/geometry/poincare_ball.py

```diff
@@ -305,3 +437,252 @@ def inner_product_matrix(self, base_point=None):
         identity = gs.eye(self.dim, self.dim)
 
         return gs.einsum('i,jk->ijk', lambda_base, identity)
+
+    def normalization_factor_init(self, variances):
+        r"""Set up function for the normalization factor.
+
+        The normalization factor is used to define Gaussian distributions
+        on the Poincaré Ball.
+
+        Parameters
+        ----------
+        variances : array-like, shape=[n_variances,]
+            An array of standard deviations.
+        normalizastion_factor_var : array-like, shape=[n_variances,]
+            An array of computed normalization factor.
+        phi_inv_var : array-like, shape=[n_variances,]
+            An array of the computed inverse of a function phi
+            whose expression is closed-form
+            :math:`\sigma\mapsto \sigma^3 \times \frac{d  }
+            {\mathstrut d\sigma}\log \zeta_m(\sigma)'
+            where :math:'\sigma' denotes the variance
+            and :math:'\zeta' the normalization coefficient
+            and :math:'m' the dimension.
+        """
+        self.variances = variances
+        self.normalization_factor_var = \
+            self.normalization_factor(variances)
+
+        cond_1 = self.normalization_factor_var.sum() != \
+            self.normalization_factor_var.sum()
+        cond_2 = self.normalization_factor_var.sum() == float('+inf')
+        cond_3 = self.normalization_factor_var.sum() == float('-inf')
+
+        if cond_1 or cond_2 or cond_3:
+            logging.warning('WARNING :\n'
```

Comment:
Great! You don't need to write "WARNING" explicitly though, logging will take care of this